projects/packages/protect-models/src/class-status-model.php

```diff
@@ -53,6 +53,13 @@ class Status_Model {
 	 */
 	public $status;
 
+	/**
+	 * Fixable threats.
+	 *
+	 * @var array
+	 */
+	public $fixable_threats = array();
```

Comment:
```suggestion
	/**
	 * List of fixable threat IDs.
	 *
	 * @var string[]
	 */
	public $fixable_threat_ids = array();
```
Minor: Could name this `$fixable_threat_ids` for clarity, and use a more specific type hint (not sure if we would want `number[]` or `string[]` 🤔